nupic/frameworks/opf/metrics.py

```diff
@@ -1459,7 +1424,62 @@ def addInstance(self, groundTruth, prediction, record = None):
 
     return self.aggregateError
 
+###################################################################################
+class MetricMulti(MetricsIface):
+  """Multi metric can combine multiple other (sub)metrics and 
+     weight them to provide combined score."""
 
+  def __init__(self, metricSpec):
+    """MetricMulti constructor using metricSpec is not allowed."""
+    raise ValueError("MetricMulti cannot be constructed from metricSpec string! "
+                     "Use MetricMulti(weights,metrics) constructor instead.")
+
+  def __init__(self, weights, metrics, id=None, window=None):
+    """MetricMulti 
+       @param weights - [list of floats] used as weights
+       @param (sub)metrics - [list of metrics] 
+       @param id - (opt) name of the metric
```

Comment:
I already have a ticket for this, https://github.com/numenta/nupic/issues/1848 I think the `minimize=` should take an ID, rather then the current string description, as the latter leads to dumb bugs when one modifies his metric in description.py but forgets to do the same modification in permutation.py's minimize=. But that's not goal of this PR. I only include id in MetricMulti because it's new and can't be created from string anyway.